src/net/sf/memoranda/ui/AgendaPanel.java

```diff
@@ -124,7 +124,7 @@ else if (d.startsWith("memoranda:tasks")) {
 							CurrentStorage.get().storeEventsManager();
 						}
 						refresh(CurrentDate.get());
-						System.out.println("agregué un sticker");
+						System.out.println("I added a sticker");

```

Comment:
Code Reviewed by ghgofort (Galen Goforth) using informal code review procedure.  --No defects found in task